pyproject.toml

```diff
@@ -276,3 +277,23 @@ doctest_optionflags = [
     "IGNORE_EXCEPTION_DETAIL",
     "ELLIPSIS",
 ]
+
+[tool.conda-lock]
+channels = [
+    "conda-forge",
+    "defaults",
+]
+platforms = [
+    "osx-arm64",
+    "linux-64",
+]
+channel-priority = "strict"
+name = "pudl-dev"
+
+[tool.conda-lock.dependencies]
+recordlinkage = {source = "pypi"}
+sphinx-autoapi = {source = "pypi"}
+pip = ">=23,<24"
+python = ">=3.11,<3.12"
+python-snappy = ">=0.6,<1"
```

Comment:
Why do we need to explicitly include `python-snappy` here instead of in our normal dependencies?

Reply:
Hmm. Here I just pulled over the packages that we needed to have in `environment.yml` because they had non-python package components that we needed to satisfy.  But I think `python-snappy` could go into the main dependencies here. I think it would mean that `libsnappydev` would need to be installed on the system if the package were getting installed with `pip` rather than `conda` via the lockfile, but that's already true.